js/skin.js

```diff
@@ -287,8 +287,42 @@ var Skin = createReactClass({
     }
   },
 
+  /**
+   * call handleTouchEnd when touchend was called on selectedScreen
```

Comment:
It doesn't explain general purpose of the method. I suggest: "Manage controls visibility and switch active screen if required, once the screen touched."
Notice how much now we would be happy to decompose this method on two:
- handleTouchEnd
- checkVrDirection (this is already implemented in controller)

Reply:
\+

Reply:
\+